### ISSUE TITLE
Controller relative routes showing up as unused

### ISSUE THREAD

**Issue by jhubert** [glebm/i18n-tasks#266]
I have the following in my en.yml:

```yml
topics:
  breadcrumbs:
    new: New Topic
```

and the following in my controller (pseudo code) :

```ruby
def new
  add_breadcrumb t('.breadcrumbs.new')
  ...
end

def create
  if object.save
    redirect_to url_for(object)
  else
    add_breadcrumb t('.breadcrumbs.new')
    render :edit
  end  
end
```

This works just fine with Rails. The proper breadcrumbs are scoped to the topics controller due to the cascade functionality of the Action View translation helper. However, the `unused` check throws up unused errors for the `.breadcrumbs.new` reference.

Do I need to some something specials to support the cascading or does i18n-tasks now support that?

**Comment by dzagozdzon** [comment]
Have you dealt with this issue somehow?

**Comment by pglombardo** [comment]
I just came across this.  I've decided to not use relative keys at all.  There's really no perfect answer to managing strings but relative keys only make it more complicated IMO.  Doubly so that this task doesn't detect their usage.

**Comment by jhubert** [comment]
I don't think we ended up solving this in any way, other than ignoring them in the YAML file.

I would agree that not using relative keys, despite being annoying and long winded, is probably better overall for maintainability.

**Comment by davidwessman** [comment]
Please use the new Prism-parser, see the README
